packages/react/src/SharedComponents/ColorPicker/components/ColorIcon.tsx

```diff
@@ -7,21 +7,27 @@ export const ColorIcon = (
     size: number | undefined;
   }>
 ) => {
+  const { classes } = createStyles({ root: {} })(undefined, {
+    name: "ColorIcon",
+  });
+
   const textColor = props.textColor || "default";
   const backgroundColor = props.backgroundColor || "default";
   const size = props.size || 16;
 
   return (
     <Box
+      className={classes.root}
       sx={(theme) => {
         return {
           backgroundColor:
             theme.colors.backgroundColors[
-              theme.other.colors.indexOf(backgroundColor)
+              theme.other.highlightColorNames.indexOf(backgroundColor)
```

Comment:
should `highlightColorNames` be part of the mantine theme? it seems unrelated tbh

Reply:
I think it's important since they should really be different for the light and dark scheme, otherwise you get contrast issues and it generally looks bad. Maybe values for light & dark should just be hardcoded though?

Reply:
I understand they should be themeable, I just don't understand the entire `indexOf` construction. Aren't we overcomplicating things?